scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/MinionProcessor.java

```diff
@@ -48,8 +48,10 @@ protected void additionalProcess(OxfordNanoporeNotificationDto onnd, IHDF5Reader
     if (reader.hasAttribute(trackingId, "device_id")) {
       onnd.setSequencerName(reader.string().getAttr(trackingId, "device_id"));
     }
-    if (reader.hasAttribute(contextTags, "sequencing_kit")) {
-      onnd.setContainerModel(reader.string().getAttr(contextTags, "sequencing_kit"));
+    if (reader.hasAttribute(contextTags, "flow_cell_product_code")) {
```

Comment:
PromethionProcessor has the exact same code, could this be moved to the process() in BaseOxfordNanoporeProcessor?

Reply:
Moved